core/src/components.d.ts

```diff
@@ -2243,7 +2243,7 @@ export namespace Components {
          */
         "animated": boolean;
         /**
-          * By default `ion-nav` animates transition between pages based in the mode (ios or material design). However, this property allows to create custom transition using `AnimateBuilder` functions.
+          * By default `ion-router-outlet` animates transition between pages based in the mode (ios or material design). However, this property allows to create custom transition using `AnimateBuilder` functions.
```

Comment:
While technically the `ion-nav` component is the responsible implementation layer for performing the transitions between pages: https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/nav/nav.tsx#L506

I do see how this documentation as a public API is confusing to implementers and the `ion-router-outlet` would be a more appropriate scope. 

I question how valuable the comment is altogether, as any consumer of the API is likely familiar with Ionic's support for iOS and Material design, as well as familiar with the routing transitions, if they are trying to access a property to customize them. The documentation could be improved with details surrounding how to build animations for routing instead of referencing existing behavior.

Reply:
I'm all for changing this comment to "Allows to create custom transition" only.

What I did here was just to replace `ion-nav` by `ion-router-outlet`. However note that the standalone `ion-router-outlet` does not animate.

Reply:
There is no reason to mention `ion-nav` (or anything related to it) here. You can use either a `ion-nav` or a `ion-router-outlet` as your outlet but there is absolutely no relation between them.

Reply:
I'm in favor of updating the comment to:
> This property allows to create custom transition using `AnimateBuilder` functions.

Also, I intended to start this comment thread in `router-outlet.tsx` instead of the auto-generated output. We'll need to update there & re-run the build command once more.

Reply:
Sounds good to me 👍